src/util/continued_fraction.js

```diff
@@ -11,6 +11,8 @@ const MAX_INT = ((1 << 31) >>> 0) - 1;
  * @returns {array} first element is n (numerator), second element is d (denominator)
  */
 export function best_r(rawNumber) {
+  BigNumber.DEBUG = true; // throws on bad number values
```

Comment:
Should probably do this globally, or unset it after this function.

Reply:
good call, fixed